src/app/therapy/page.js

```diff
@@ -1,46 +1,15 @@
-'use client';
-
-import React, { useState } from 'react';
-import {
-  useListTherapies,
-  useCreateTherapy,
-  useDeleteTherapy,
-} from '../_hooks';
-
-export default function Therapies() {
-  const [price, setPrice] = useState('');
-  const { data: therapies } = useListTherapies();
-  const { mutate: createTherapy } = useCreateTherapy();
-  const { mutate: deleteTherapy } = useDeleteTherapy();
-
-  const onAdd = () => {
-    createTherapy({ price: Number(price) }, { onSuccess: () => setPrice('') });
-  };
-
-  const onDelete = (id) => {
-    deleteTherapy({ id });
-  };
-
-  return (
-    <div style={{ margin: '10rem' }}>
-      <input
-        type="number"
-        value={price}
-        onChange={e => setPrice(e.target.value)}
-      />
-      <button type="button" onClick={() => onAdd()}>
-        Add
-      </button>
-      <div>
-        {therapies?.map(it => (
-          <div key={it.id} style={{ display: 'flex', gap: '2rem' }}>
-            <div>{`${it.id}:${it.price}`}</div>
-            <button type="button" onClick={() => onDelete(it.id)}>
-              Delete
-            </button>
-          </div>
-        ))}
-      </div>
-    </div>
-  );
+import React from 'react';
+import Therapies from '../_components/Therapies';
+
+export const metadata = {
```

Comment:
Have a suggestion, though it's up to you. We can create '/src/app/_const/metadata.js to store all the meta in one place.

`
const metadata = {
  admin: {
    index: {
      title: 'Адмінка',
      description: 'Адмінка сайту Mayak',
    },
  },
  app: {
    index: {
      title: 'Головна сторінка',
      description: 'Пошук психологічної допомоги у м.Львів',
    },
    specislits: {
      title: '...',
      description: '...',
    }
  },
};

export default metadata;
`

Reply:
This could be nice if all of the metadata was static. However, there can be cases when the metadata should be generated dynamically (using generateMetadata function).
for example, when using certain search params, the metadata can be generated to represent specified search.
http://mayak.com/specialist?district=frankivsky
this could have 
{
title: "Пошук спеціаліста в Франківському районі."
}
http://mayak.com/specialist/certain_id
this could have 
{
title: "Степан Журавчак" // based on specialists name 
}